### PR TITLE
Banner js

### PR DESCRIPTION
**Pull request by tommyneu**
Moved Banner javascript code from html template to its own file and reworked where it would be compiled

### CODE REVIEW

File: wdn/templates_5.3/js-src/banner.babel.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+            banner.innerHTML = messageContent;
+            if (skipNav) {
+                skipNav.parentNode.insertBefore(banner, skipNav.nextSibling);
+            } else {
+                document.body.prepend(banner);
+            }
+        }
+    }
+}
+displayWDNNoticeBanner();
```
Comment (skoolbus39) [inline wdn/templates_5.3/js-src/banner.babel.js:45]: No newline at end of file

**Review by skoolbus39 — CHANGES_REQUESTED** [review]

File: wdn/templates_5.3/js-src/banner.babel.js (hunk 1 above)
Comment (tommyneu) [reply wdn/templates_5.3/js-src/banner.babel.js]: added

**Review by skoolbus39 — APPROVED** [review]

File: wdn/templates_5.3/js-src/banner.babel.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
+// WDN Notice Banner Display
+function displayWDNNoticeBanner() {
+    var bannerEnabled = true;
+    if (bannerEnabled) {
+        var messageContent = false;
+        var messageSessionKey = 'wndNoticeMessage';
```
Comment (ericras) [inline wdn/templates_5.3/js-src/banner.babel.js:6]: we have a typo here: 'wnd' that's never gotten fixed. let's change this.

File: wdn/templates_5.3/js-src/banner.babel.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        var sessionContent = sessionStorage.getItem(messageSessionKey);
+
+        // clear saved message every five minutes
+        setInterval(function(){ sessionStorage.removeItem(messageSessionKey); }, 300000);
```
Comment (ericras) [inline wdn/templates_5.3/js-src/banner.babel.js:10]: why don't we bump this up to like 15 (?) minutes

File: wdn/templates_5.3/js-src/banner.babel.js (hunk 2 above)
Comment (tommyneu) [reply wdn/templates_5.3/js-src/banner.babel.js:6]: fixed

File: wdn/templates_5.3/js-src/banner.babel.js (hunk 3 above)
Comment (tommyneu) [reply wdn/templates_5.3/js-src/banner.babel.js:10]: i bumped it to 15 and changed how to calcuates the minutes to milliseconds for easier readability

**Pull request merged**